sensors/mqtt_sensor.py

```diff
@@ -32,10 +32,17 @@ def __init__(self, sensor_service, config=None):
         self._ssl_cacert = self._config.get('ssl_cacert', None)
         self._ssl_cert = self._config.get('ssl_cert', None)
         self._ssl_key = self._config.get('ssl_key', None)
+        self._transport = self._config.get('transport', 'tcp')
 
     def setup(self):
         self._logger.debug('[MQTTSensor]: setting up sensor...')
 
+        # if no topics are configured for subscription, then don't run
+        # the sensor
+        if not self._subscribe:
+            raise ValueError(
```

Comment:
I wasn't sure if this was the "preferred" way to handle preventing sensor startup but looking at a few other sensor examples on the stackstorm exchange, this seemed to be the most common implementation.